### DISCUSSION TITLE
EU-SILC_CS v0.0

### DISCUSSION THREAD

**Discussion by alzbetab** [Open-Family-Policy-Platform/EU-SILC_CS#2] [General]
This is a beta version of the Open Family Policy Program for EU-SILC cross-sectional data. It covers maternity, paternity and parental leave policies in 28 European countries for the years 2018 and 2019.

<hr /><em>This discussion was created from the release <a href='https://github.com/Open-Family-Policy-Platform/EU-SILC_CS/releases/tag/v0.0'>EU-SILC_CS v0.0</a>.</em>

**Comment by pospjan** [comment]
It's great, thank you!
